nwb_conversion_tools/conversion_tools.py

```diff
@@ -40,7 +40,7 @@ def get_default_nwbfile_metadata():
     metadata = dict(
         NWBFile=dict(
             session_description="no description",
-            session_start_time=datetime(1970, 1, 1),
+            session_start_time=datetime(1970, 1, 1).isoformat(),
```

Comment:
@luiztauffer Why is this change necessary? @bendichter can correct me if I'm wrong, but the pynwb module specifically acts like the session_start_time should be a datetime object, not a string: https://github.com/NeurodataWithoutBorders/pynwb/blob/66ef3abeaa5bd9200ad926ddb82c0d53eba11281/src/pynwb/file.py#L207

Though the NWB Schema itself seems to specify ISO (https://nwb-schema.readthedocs.io/en/latest/format.html#type-specifications), I therefore assume that pynwb handles the translation.

Reply:
json schemas accept times only as strings, this needs to be a string to pass validation